packages/vite/src/plugins/vite-plugin-rsc-routes-auto-loader.ts

```diff
@@ -1,8 +1,8 @@
 import path from 'path'
 
-import generate from '@babel/generator'
-import { parse as babelParse } from '@babel/parser'
-import traverse from '@babel/traverse'
+import bgen from '@babel/generator'
```

Comment:
*STUCK - problem 2*
I'm having to do these weird imports because the babel packages are CJS until they release Babel 8. See this issue here: https://github.com/babel/babel/issues/16599

The way I have it written _does_ work (for code, not for types), but feels like an ugly hack. 

It's done this way because I need to check `babelGenerate.default` when it's actually running build in Vite on a user's project. Without .default it doens't find traverse or generate functions.

BUT for vitest, the import is _not_ wrapped in default. So

```js
import btrav from '@babel/traverse'

console.log(btrav)
// In vitest something like:
//  [Function]

// When building in app:
// { default: [Function] }

```

Reply:
Solved ✅✅✅✅

This was caused by https://vitest.dev/config/#deps-interopdefault
I think it's better to have it this way because it aligns with when the code is actually running.